core/tests/protractor/editorAndPlayer.js

```diff
@@ -159,6 +208,23 @@ describe('Full exploration editor', function() {
         forms.toRichText('Parent Exploration Content'));
       explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'Correct');
       browser.waitForAngular();
+
+      browser.get('/search/find?q=');
+      element.all(by.css(
+        '.protractor-test-collection-summary-tile-title')).first().click();
+      element.all(by.css(
+        '.protractor-test-collection-exploration')).first().click();
+      explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'Incorrect');
+      general.waitForSystem();
+      explorationPlayerPage.clickConfirmRedirectionButton();
+      browser.waitForAngular();
+      // Checking the current url to see if collection_id is present in it.
```

Comment:
Nit: Checking --> Check

(it's an imperative, not an ongoing process)

Otherwise lgtm! Thanks!

Reply:
Done, thanks!